File: server/src/main/java/de/ingrid/igeserver/exports/iso19115/Thesaurus.java
package de.ingrid.igeserver.exports.iso19115;

import java.util.List;

public class Thesaurus {

    public String name;

    public String date;

    public List<Keyword> keywords;

    public Thesaurus(String name, String date, List<Keyword> keywords) {
        this.name = name;
        this.date = date;
        this.keywords = keywords;
    }

}

File: server/src/main/java/de/ingrid/igeserver/exports/iso19115/Thesaurus.kt
package de.ingrid.igeserver.exports.iso19115

data class Thesaurus(
    var name: String? = null,
    var date: String? = null,
    var link: String? = null,
    var showType: Boolean = true,
    var keywords: List<Keyword> = emptyList()
)
